openshift/odo
# odo url  new flow

## What is the purpose of this change? What does it change?

* Modifying `odo url` as per new work flow
* Make `--port` flag compulsory

## Was the change discussed in an issue?
fixes #1389

## How to test changes?

* `odo url create --port 8080`  (it will make entry to .odo/config.yml)
* `odo push` should create URL on cluster
* `odo url list` (it will list urls which are created on cluster and also urls which has entry in config file but not on the cluster)
* `odo url delete <url_name>`
